src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/UserAccountManager.java

```diff
@@ -1122,10 +1122,6 @@ public RegisteredUserDTO createUserObjectAndSession(
       userToSave.setRegisteredContextsLastConfirmed(new Date());
     }
 
-    if (userToSave.getTeacherPending() == null) {
```

Comment:
Is there any harm in leaving this as it was? We definitely need this check in `PgUsers`, but leaving it here it also means that this method will create a valid user. Thoughts?

Reply:
I can put it back in if that would be preferable? Just seemed redundant with the new check. 🤷

Reply:
It's just in case the get teacher pending flag is read between here and where the new check is. By removing this, the variable can potentially be `null` in places where it wasn't before.